iss.py

```diff
@@ -1,11 +1,101 @@
 #!/usr/bin/env python
 
-__author__ = '???'
+__author__ = 'katran009 and google'
 
+import json
+import urllib.request
```

Comment:
I recommend to use the `requests` library as mentioned in the readme -- it has a much better api interface and it includes json data handling and parsing built in.  No need to import json separately